lwr_controllers/src/dynamic_sliding_mode_controller.cpp

```diff
@@ -157,7 +41,7 @@ namespace lwr_controllers
 		k_.resize(kdl_chain_.getNrOfJoints());
 
 		sub_command_ = nh_.subscribe("command_configuration", 1, &DynamicSlidingModeController::command_configuration, this);
```

Comment:
You mean the standard controllers? Because all lwr_controllers use "command_configuration" as name